packages/react-native/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "include": ["src"],
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "target": "ES6",
+    "esModuleInterop": true,
+    "module": "NodeNext",
+    "isolatedModules": true,
+    "jsx": "react-jsx",
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "NodeNext",
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true
+  },
```

Comment:
please check the suggested TS config for RN: https://github.com/tsconfig/bases?tab=readme-ov-file#react-native-tsconfigjson

Reply:
Will take a look at this, not sure if it will be needed. 🙏